app.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, render_template, session, redirect, url_for
+from flask_wtf import FlaskForm
+from wtforms import SubmitField
+from wtforms import IntegerField
+from wtforms.validators import DataRequired
+from flask_bootstrap import Bootstrap
+from coil import normalcoil
+
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'jakis ciag znakow'
```

Comment:
I don't recognize this language. Please use human language (english)

Reply:
:D